django_tables2/rows.py

```diff
@@ -71,12 +71,20 @@ def table(self):
         '''
         return self._table
 
+    def get_even_odd_css_class(self):
+        '''
+        Return css class `even` or `odd`
+        '''
+        if next(self._table._counter) % 2 == 0:
```

Comment:
Not sure what you mean by '25% for 10^6 rows', but 10^6 rows is a very unrealistic scenario: pagination should be used for datasets for more than about 100 rows.